include/click/algorithm.hh

```diff
@@ -90,5 +90,8 @@ struct less {
     }
 };
 
+#define click_min(a, b) ((a) < (b) ? (a) : (b))
+#define click_max(a, b) ((a) > (b) ? (a) : (b))
+
```

Comment:
:cry:
Must we? You only use click_min once and you use click_max not at all. Let's scotch this.